specification/azureactivedirectory/resource-manager/readme.md

```diff
@@ -26,14 +26,23 @@ These are the global settings for the azureactivedirectory.
 
 ``` yaml
 openapi-type: arm
-tag: package-preview-2020-07
+tag: package-preview-2017-04
 ```
 
+
+### Tag: package-preview-2017-04
+
+These settings apply only when `--tag=package-preview-2017-04` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2017-04'
+input-file:
+  - Microsoft.Aadiam/preview/2017-04-01-preview/azureactivedirectory.json
```

Comment:
Introducing an old API version is not friendly for customers.  Especially a preview version of an existing stable API version.  Please use a recent date for the new preview API version